src/components/events/Event.js

```diff
@@ -1,7 +1,39 @@
+"use client";
 import React from "react";
-
-const Event = () => {
-  return <div>Event</div>;
+// style the component here
+const Event = ({ title, location, start, color, description }) => {
+  return (
+    <div className={"flex rounded-lg flex-row font-urbanist p-4"}>
+      <div className={color}>
+        <div className={"flex text-center text-winc-white flex-col p-5"}>
```

Comment:
remove all the curly braces from classNames that do not have a variable passed in.